frontend/src/container/Dashboard.jsx

```diff
@@ -413,7 +413,7 @@ class Dashboard extends React.Component {
                               hexcolor={app.hexcolor}
                               label={app.label}
                               faIcon={app.faIcon}
-                              // @fixme Côme - 2018/09/12 - trad key bellow is a little hacky. The creation label comes from api but since there is no translation in backend
+                              // Côme - 2018/09/12 - trad key bellow is a little hacky. The creation label comes from api but since there is no translation in backend
                               // every files has a 'externalTradList' array just to generate the translation key in the json files through i18n.scanner
```

Comment:
Can you use the normalized format for this note : `TODO CH 2018-09-12: bla bla bla` ? Thank's.

Reply:
done

Reply:
remark: TODO means you would like to fix it later, if it's just a dirty part of code but you do not intend to fix it, then a HACK keyword would be better.

Reply:
ok, "TODO" is the right word then